pkg/operator/nodecadaemon_test.go

```
@@ -189,7 +196,135 @@ func newNodeCATestSetup(t *testing.T, applyError error) (*NodeCADaemonController
+				NumberMisscheduled:     1,
+			},
+			metadataGeneration:            8,
+			lastSettledObservedGeneration: 8,
+			expectProgressing:             operatorv1.ConditionTrue,
+			expectSettledGeneration:       8,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)